NutritionNest/Views/Onboarding/OnboardingView.swift

```diff
@@ -2,41 +2,42 @@
 //  ContentView.swift
 //  Loginpage
 //
-//  Created by LiTianyi on 2023/9/27.
+//  Created by Oliver on 2023/9/27.
 //
 
 import SwiftUI
 
 struct OnboardingView: View {
     var body: some View {
         VStack {
-            Text(Localization.Onboarding.welcome).font(.system(size: 26)).bold().padding()
-            Image("logo")
+            Text(Localization.Onboarding.welcome).font(.custom("Inter", size: 26)).bold().padding()
```

Comment:
Here, you should use the custom extension I made for the inter font. Like so:
`.font(.custom.inter.font(size: 15)).bold().padding()`